src/main/resources/templates/smallfull/balanceSheet.html

```diff
@@ -7,7 +7,7 @@
     <head>
         <title>Balance Sheet</title>
     </head>
-
+<body>
```

Comment:
Should this be here? The [baseLayout](https://github.com/companieshouse/company-accounts.web.ch.gov.uk/blob/develop/src/main/resources/templates/layouts/baseLayout.html#L18) is responsible for holding the body, and every page we develop has the [content](https://github.com/companieshouse/company-accounts.web.ch.gov.uk/blob/develop/src/main/resources/templates/layouts/baseLayout.html#L26) layout fragment to be added to the body